js/gateio.js

```diff
@@ -570,8 +570,8 @@ module.exports = class gateio extends Exchange {
         // :param params['type']: 'spot', 'margin', 'future' or 'delivery'
         // :param params['settle']: The quote currency
         const [ type, query ] = this.handleMarketTypeAndParams ('fetchMarkets', undefined, params);
-        const spot = (type === 'spot');
         const margin = (type === 'margin');
+        const spot = (type === 'spot') || margin;
```

Comment:
On almost all the other exchanges, if margin is true, spot is also true, so I thought it should be that way on gateio also.